src/encoded/schemas/pathology_report.json

```diff
@@ -0,0 +1,604 @@
+                  "Absent",
+                  "Not applicable",
+                  "Not available"
+              ]
+      },
+      "micro_gerota": {
+          "title": "Gerotas Fascia Involvement",
+          "description": "Tumor extension beyond Gerotas Fascia, identified microscopically",
+          "type": "string",
+          "enum": [
+                  "Present",
+                  "Absent",
+                  "Not applicable",
+                  "Not available"
+              ]
+      },
+      "micro_pelvaliceal": {
+          "title": "Pelvicaliceal Involvement",
+          "description": "Tumor extension into pelvicalyceal, identified microscopically",
+          "type": "string",
+          "enum": [
+                  "Present",
+                  "Absent",
+                  "Not applicable",
+                  "Not available"
+              ]
+      },
+      "t_stage": {
+          "title": "pT Stage",
+          "description": "Pathological T stage, size primary tumor",
+          "type": "string",
+          "enum": [
+                  "pT1",
+                  "pT1a",
+                  "pT1b",
+                  "pT2",
+                  "pT2a",
+                  "pT2b",
+                  "pT3",
+                  "pT3a",
+                  "pT3b",
+                  "pT3c",
+                  "pT4",
+                  "pTX"
+              ]
+      },
+      "n_stage": {
+          "title": "pN Stage",
+          "description": "Pathological N stage, nodal involvement",
+          "type": "string",
+          "enum": [
+                  "pN0",
+                  "pN1",
+                  "pN2",
+                  "pNX",
+                  "Not applicable",
+                  "Not available"
+              ]
+      },
+      "examined_lymph_nodes": {
+          "title": "Examined Lymph Nodes",
+          "description": "Number of lymph nodes examined",
+          "type": "number"
+      },
+      "positive_lymph_nodes": {
+          "title": "Positive Lymph Nodes",
+          "description": "Number of lymph nodes positive",
+          "type": "number"
+      },
+      "m_stage": {
+          "title": "pM Stage",
+          "description": "Pathological M stage, nodal involvement",
+          "type": "string",
+          "enum": [
+                  "pM0",
+                  "pM1",
+                  "Not applicable",
+                  "Not available"
+              ]
+      },
+      "ajcc_version": {
+        "title": "AJCC Version",
+        "description": "AJCC Version used for staging",
+        "type": "string",
+        "enum": [
+                "6th edition",
+                "7th edition"
+            ]
+      },
+      "ajcc_tnm_stage": {
+        "title": "AJCC TNM Stage",
+        "description": "AJCC TNM Stage",
+        "type": "string",
+        "enum": [
+                "1",
+                "2",
+                "3",
+                "4"
+            ]
+      },
+      "ajcc_p_stage": {
```

Comment:
"ajcc_p_stage"( "title": "AJCC pT Stage") seems to be a duplicated property.

Reply:
Not matters currently I think.

Reply:
We should go over all and see what are redundant